src/TisaBackend/TisaBackend.Domain/Models/AirPlaneType.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace TisaBackend.Domain.Models
+{
+    public class AirPlaneType
```

Comment:
AirPlaneType == planeModel from Word?

Reply:
Yes (I didn't want to confuse with Models dir)